### PR TITLE
query multiple tables

### PR DESCRIPTION
**Pull request by atatm**
### changes after the second code review
- [ ] remove the live demo in the readme file as the code reviewer recommended

-- [ ] changes after the first code review
- create a license 
- update query.sql as recommended
Screenshot (41)

Screenshot (39)
Screenshot (34)
Screenshot (35)
Screenshot (36)
Screenshot (37)
Screenshot (38)

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -46,3 +46,38 @@ SELECT name,AVG(weight_kg) FROM animals GROUP BY name;
...
+GROUP BY S.name;
+
+
+SELECT A.name,O.full_name
+FROM animals A
+JOIN species S ON A.species_id = S.id
+JOIN owners O ON A.owner_id = O.id
+WHERE S.name = 'Digimon' AND O.full_name = 'Jennifer Orwell';
+
+
```
Comment (rloterh) [inline queries.sql:78]: - [ ] Queries for _Who owns the most animals?_ is missing! Kindly ensure to include them, which should be similar to:

```sql
SELECT owners.full_name, COUNT(owners.full_name) FROM animals
   LEFT JOIN owners
   ON animals.species_id = owners.id
   GROUP BY owners.full_name
   ORDER BY COUNT(owners.full_name) DESC
```

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Alelign, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦   🏃 

## Highlights 👍  
- Good job so far implementing queries for multiple tables!

## Required Changes ♻️
- [ ] Kindly ensure to remove redundant demo links in your ReadMe.
- [ ] MIT license link in your ReadMe is broken, kindly endeavor to fix it, and remove the guide note below it.

_Kindly check the other inline comments under the review._

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @atatm 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
* Well written PR description 👍 
* Proper use of Gitflow 👍 

 ## Required Changes ♻️
* As mentioned by the previous CR , kindly remove the live demo link on your readme file as this project doesn't have one 
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidosky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @atatm ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️
- Good commit messages ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**